docs/sdk/serverless-signing/quick-start.md

```diff
@@ -7,557 +7,188 @@ import TabItem from '@theme/TabItem';
-    ],
-});
-
-// output:
-{
-  pkp: {
-      tokenId: string;
-      publicKey: string;
-      ethAddress: string;
-  };
-  tx: ethers.ContractReceipt;
-}
-```
-
-You should now have successfully minted a PKP! You can verify that the PKP has the necessary permissions for signing by calling the following function:
-
-```jsx
-import { LitAuthClient } from '@lit-protocol/lit-auth-client';
-
-const authId = await LitAuthClient.getAuthIdByAuthMethod(authMethod);
-const scopes = await contractClient.pkpPermissionsContract.read.getPermittedAuthMethodScopes(
-  mintInfo.pkp.tokenId,
-  AuthMethodType.EthWallet,
-  authId,
-  3
-);
-
-const signAnythingScope = scopes[1];
-const personalSignScope = scopes[2];
-```
-
-Additional Demos: 
-
-1. **[Minting a PKP with an auth method and permitted scopes (Easy)](https://github.com/LIT-Protocol/js-sdk/blob/feat/SDK-V3/e2e-nodejs/group-contracts/test-contracts-write-mint-a-pkp-and-set-scope-1-2-easy.mjs)**
-2. **[Minting a PKP with an auth method and permitted scopes (Advanced)](https://github.com/LIT-Protocol/js-sdk/blob/feat/SDK-V3/e2e-nodejs/group-contracts/test-contracts-write-mint-a-pkp-and-set-scope-1-advanced.mjs)**
-3. **[Minting a PKP using social login](../../user-wallets/pkps/minting/via-social.md)**
-
-## Mint Capacity Credits and Delegate Usage
-
-In order to execute a transaction with Lit, you’ll need to reserve capacity on the network using Capacity Credits. These allow holders to reserve a set number of requests (requests per second) over a desired period of time (i.e. one week). You can mint a Capacity Credit NFT using the `contracts-sdk` in a couple of easy steps. 
-
-The first step is to initialize a signer. This should be a wallet controlled by your application and the same wallet you’ll use to mint the Capacity Credit NFT:
-
-```jsx
-import { LitNetwork } from "@lit-protocol/constants";
-
-const walletWithCapacityCredit = new Wallet("<your private key or mnemonic>");
-
-let contractClient = new LitContracts({
-  signer: dAppOwnerWallet,
-  network: LitNetwork.Datil,
-});
-
-await contractClient.connect();
-```
-
-After you’ve set your wallet, your next step is to mint the NFT:
-
-```jsx
-// this identifier will be used in delegation requests. 
-const { capacityTokenIdStr } = await contractClient.mintCapacityCreditsNFT({
-  requestsPerKilosecond: 80,
-  // requestsPerDay: 14400,
-  // requestsPerSecond: 10,
-  daysUntilUTCMidnightExpiration: 2,
-});
-```
-
-In the above example, we are configuring 2 properties:
-
-- `requestsPerDay` - How many requests can be sent in a 24 hour period.
-- `daysUntilUTCMidnightExpiration` - The number of days until the nft will expire. expiration will occur at `UTC Midnight` of the day specified.
+</p>
+</details>
 
-Once you mint your NFT you will be able to send X many requests per day where X is the number specified in `requestsPerDay`. Once the `Capacity Credit` is minted the `tokenId` can be used in delegation requests.
+## Defining the Lit Action
```

Comment:
perhaps use "writing" or "creating" here

Reply:
ahhh, or deploying as that seems to be the context?

Reply:
Changed to `storing`